internal_transfer_with_agreed_amount/security/res_groups_security.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
-    <record id="allow_confirm_internal_transfer_with_agreed_amount" model="res.groups">
+    <record id="module_category_usability" model="res.groups">
```

Comment:
This change is not correct, this is not a view

Reply:
Change undone.